.github/workflows/doc.yml

```diff
@@ -0,0 +1,73 @@
+name: Documentation
+
+on:
+  push:
+    paths:
+    - 'config/**'
+    - '!config/stack/ttn-lw-stack.yml'
+    - 'doc/**'
```

Comment:
We should probably add the API as that reference is generated.

Reply:
Yeah, but not from `/api` currently. See https://github.com/TheThingsIndustries/docker-protobuf/issues/33.